functions/_fzf_complete.fish

```diff
@@ -0,0 +1,178 @@
+# External limitations:
```

Comment:
to keep with the naming convention of the other search functions, this should be named "_fzf_search_completions.fish"

Reply:
Good point, all occurrences have been renamed accordingly